### PR TITLE
chore(blocks): refine linked doc toolbar

### PR DESCRIPTION
**Pull request by fundon**
Closes: [BS-584](https://linear.app/affine-design/issue/BS-584/linked-doc-toolbar)

### CODE REVIEW

**Comment by fundon** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7483?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#7483** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7483?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#7479** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7479?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7475** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7475?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: tests/edgeless/linked-doc.spec.ts [hunk 1]
```diff
@@ -42,13 +42,6 @@ test.describe('note to linked doc', () => {
     await waitNextFrame(page, 200);
     const embedSyncedBlock = page.locator('affine-embed-synced-doc-block');
     assertExists(embedSyncedBlock);
-
-    await triggerComponentToolbarAction(page, 'linkedDocInfo');
```
Comment (Saul-Mirone) [inline tests/edgeless/linked-doc.spec.ts:46]: If you want to remove these tests, you should add new one to protect your feature.

**Review by Saul-Mirone — CHANGES_REQUESTED** [review]

**Review by Svaney-ssman — COMMENTED** [review]
UI Issues:
1. both these two button status are hover, the default display should not have background color
CleanShot 2024-07-04 at 17 27 21@2x

2. the font wight of menu is incorrect
CleanShot 2024-07-04 at 17 30 24@2x
CleanShot 2024-07-04 at 17 31 10@2x
CleanShot 2024-07-04 at 17 33 07@2x

3. the reload button is missing

**Review by Saul-Mirone — APPROVED** [review]
LGTM

**Review by Saul-Mirone — COMMENTED** [review]
CI fails need to be fixed.

**Comment by fundon** [comment]
> CI fails need to be fixed.

Maybe it will be better to check it in https://github.com/toeverything/blocksuite/pull/7520.

**Comment by Saul-Mirone** [comment]
> > CI fails need to be fixed.
> 
> Maybe it will be better to check it in #7520.

In graphite you need to make sure every pr pass the ci check.

**Review by doodlewind — APPROVED** [review]

**Pull request merged**